server/service/src/invoice_line/outbound_shipment_line/update/generate.rs

```diff
@@ -115,7 +114,7 @@ fn generate_line(
         item_code,
         stock_line_id: Some(stock_line_id),
         total_before_tax,
-        total_after_tax,
+        total_after_tax: total_after_tax(total_before_tax, tax),
```

Comment:
total_after_tax should be calculated at the end after calculating total_before_tax, right?

Reply:
[17e41f8](https://github.com/openmsupply/open-msupply/pull/441/commits/17e41f8d21a1ae9ed9eb55f59dae07b9cf5f3840)

Reply:
[c56dbde](https://github.com/openmsupply/open-msupply/pull/441/commits/c56dbde13527207189fb6bc64515b8e3ae0bedc7)